src/lib/io/sol-iio.c

```diff
@@ -544,12 +544,27 @@ channel_get_pure_name(struct sol_iio_channel *channel)
 {
```

Comment:
Please, also update this function commentary, to include this 'both' and 'ir' thing.
That said, shouldn't you also include 'uv' suffix? It appears on kernel doc...